locations/spiders/providence_health_services.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+import json
+import re
+import scrapy
+from locations.items import GeojsonPointItem
+from locations.hours import OpeningHours
+
+
+class Providence_health_servicesSpider(scrapy.Spider):
```

Comment:
Can you use a different name?

```suggestion
class ProvidenceHealthServicesSpider(scrapy.Spider):
```